src/OpenLoco/Vehicles/VehicleHead.cpp

```diff
@@ -2388,14 +2388,14 @@ namespace OpenLoco::Vehicles
                         continue;
                     }
 
-                    industry->var_199[i] = std::min<uint32_t>(industry->var_199[i] + car->cargoQty, std::numeric_limits<uint16_t>::max());
-                    industry->var_199[i] = std::min<uint32_t>(industry->var_18D[i] + car->cargoQty, std::numeric_limits<uint16_t>::max());
+                    industry->var_199[i] = static_cast<uint16_t>(std::min<uint32_t>(industry->var_199[i] + car->cargoQty, std::numeric_limits<uint16_t>::max()));
+                    industry->var_18D[i] = static_cast<uint16_t>(std::min<uint32_t>(industry->var_18D[i] + car->cargoQty, std::numeric_limits<uint16_t>::max()));
```

Comment:
This seems tautological. Are you trying to prevent overflow? With `uint32_t` as the type, there's no way for it to ever choose uint16_max, though…

Reply:
Yeah we don't want wraparound on these values. So 0xffff + 1 will be 0xffff. I'm going to write a helper function for this

Reply:
Okay, but that's not going to work here, is it? Wouldn't 0xffff + N just overflow to 0 and beyond? In which case, `std::max` might be what you're looking to use?